app/_how-tos/use-semantic-prompt-guard.md

```diff
@@ -0,0 +1,274 @@
+          model:
+            name: text-embedding-3-small
+            provider: openai
+        search:
+          threshold: 0.7
+        vectordb:
+          strategy: redis
+          distance_metric: cosine
+          threshold: 0.5
+          dimensions: 1024
+          redis:
+            host: ${redis_host}
+            port: 6379
+        rules:
+          match_all_conversation_history: true
+          allow_prompts:
+            - Network troubleshooting and diagnostics
+            - Cloud infrastructure management (AWS, Azure, GCP)
+            - Cybersecurity best practices and incident response
+            - DevOps workflows and automation
+            - Programming concepts and language usage
+            - IT policy and compliance guidance
+            - Software development lifecycle and CI/CD
+            - Documentation writing and technical explanation
+            - System administration and configuration
+            - Productivity and collaboration tools usage
+          deny_prompts:
+            - Hacking techniques or penetration testing without authorization
+            - Bypassing software licensing or digital rights management
+            - Instructions on exploiting vulnerabilities or writing malware
+            - Circumventing security controls or access restrictions
+            - Gathering personal or confidential employee information
+            - Using AI to impersonate or phish others
+            - Social engineering tactics or manipulation techniques
+            - Guidance on violating company IT policies
+            - Content unrelated to work, such as entertainment or dating
+            - Political, religious, or sensitive non-work-related discussions
+variables:
+  openai_api_key:
+    value: $OPENAI_API_KEY
+  redis_host:
+    value: $YOUR_REDIS_HOST
+{% endentity_examples %}
+
+
+## Validate configuration
+
+Once the AI Semantic Prompt Guard plugin is configured, you can test different kinds of prompts to make sure the guardrails are working. Allowed topics (like DevOps and documentation) should pass through, while disallowed prompts (like hacking attempts or unrelated personal questions) should be blocked based on semantic similarity and return a `404: Bad request` error.
+
+
+{% navtabs "semantic-prompt-guard-tests" %}
+{% navtab "Allowed: DevOps and automation" %}
+
+These prompts are within allowed IT topics like automation, DevOps, and CI/CD:
+
+{% validation request-check %}
+url: /anything
+headers:
+  - 'Content-Type: application/json'
+  - 'Authorization: Bearer $DECK_OPENAI_API_KEY'
+body:
+  messages:
+    - role: user
+      content: What are the best tools for DevOps?
+{% endvalidation %}
+
+{% validation request-check %}
+url: /anything
+headers:
+  - 'Content-Type: application/json'
+  - 'Authorization: Bearer $DECK_OPENAI_API_KEY'
+body:
+  messages:
+    - role: user
+      content: What are best practices in CI/CD automation?
+{% endvalidation %}
```

Comment:
```suggestion
```
Same feedback as a previous PR, I think having one example per tab is sufficient and will make the rendered formatting cleaner.